### PR TITLE
Fixes: #1370 Set picture in post as their natural size

### PR DESCRIPTION
**Pull request by yuanLeeMidori**
## Issue This PR Addresses

This PR takes over the old PR #1373 
Fixes: #1370 

Please leave comments about this css styling, thank you!

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

This is the current picture styling, which is not the natural size of the original blog.
Screenshot from 2021-02-18 17-30-53

This is using the natural size of the original blog post.
Screenshot from 2021-02-18 17-31-25

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by humphd** [comment]
I'm not sure if making this a block element (i.e., `flex` is just like saying `block` here, since the `img` has no children) will work for all images.  What about inline images?

I agree it fixes the problem of having smaller images centered and use margins vs. expanding.

File: src/frontend/next/src/styles/telescope-post-content.css [hunk 1]
```diff
@@ -19,7 +19,7 @@
   max-width: 100%;
   padding-top: 1.5rem;
   padding-bottom: 1rem;
-  width: 100%;
+  display: flex;
 }
```
Comment (foxviet) [inline src/frontend/next/src/styles/telescope-post-content.css:23]: with ```max-width: 100%```
Then you just need ```height: auto```

**Review by foxviet — CHANGES_REQUESTED** [review]

File: src/frontend/next/src/styles/telescope-post-content.css (hunk 1 above)
Comment (yuanLeeMidori) [reply src/frontend/next/src/styles/telescope-post-content.css:23]: Thank you for the comment! But what exactly does `height: auto` do in this case? I can't see it affect the image size (correct me if I'm wrong)

**Comment by yuanLeeMidori** [comment]
So in this PR, I basically removed the `width: 100%` to avoid the picture being expanded to larger size. However, I'm not sure if this is gonna affect the image overflow issue.

Please leave comment if you have any thought, thanks!

**Comment by birtony** [comment]
@yuanLeeMidori  I think you should center it 
image

**Comment by yuanLeeMidori** [comment]
followup: #1791

**Review by tonyvugithub — DISMISSED** [review]

**Review by birtony — APPROVED** [review]

**Review by humphd — APPROVED** [review]

**Review by birtony — APPROVED** [review]

**Review by tonyvugithub — APPROVED** [review]

**Review by chrispinkney — APPROVED** [review]

**Review by PedroFonsecaDEV — APPROVED** [review]

**Review by UsernameHasBeenChanged100Times — APPROVED** [review]

**Review by HyperTHD — APPROVED** [review]

**Pull request merged**